editor/store/selectors.js

```diff
@@ -1352,7 +1353,13 @@ export const getEditedPostContent = createSelector(
 			return edits.content;
 		}
 
-		return serialize( getBlocks( state ) );
+		const blocks = getBlocks( state );
+
+		if ( blocks.length === 1 && blocks[ 0 ].name === getUnknownTypeHandlerName() ) {
```

Comment:
It's not evident why we're doing this. An inline code comment would have been appropriate.